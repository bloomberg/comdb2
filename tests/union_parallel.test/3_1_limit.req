select c as col from t2 union all select e from t3 limit 2
