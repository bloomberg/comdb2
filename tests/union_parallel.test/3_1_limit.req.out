(c=11)
(c=22)
